spec.html

```diff
@@ -36719,8 +36709,7 @@
                     1. Let _genvRec_ be _genv_'s EnvironmentRecord.
                     1. Let _benv_ be the running execution context’s LexicalEnvironment.
                     1. Let _benvRec_ be _benv_'s EnvironmentRecord.
-                    1. Let _fobj_ be _benvRec_.GetBindingValue(_F_, *false*).
-                    1. Assert: _fobj_ is never an abrupt completion.
+                    1. Let _fobj_ be ! _benvRec_.GetBindingValue(_F_, *false*).
                     1. Let _status_ be ? _genvRec_.SetMutableBinding(_F_, _fobj_, *false*).
```

Comment:
this `_status_` is never used, should it be `Perform ? _genvRec_.SetMutableBinding(_F_, _fobj_, *false*).`?